src/app/core/tools/InteractionTool.ts

```diff
@@ -37,6 +38,7 @@ export class InteractionTool extends DefaultTool {
 
         const worldMousePos = camera.getWorldPos(input.getMousePos());
         const obj = this.findObject(worldMousePos, info);
+        if (!obj) return super.onEvent(event, info);
```

Comment:
? this seems like new behavior, why are you returning `super.onEvent`?

Reply:
Because `this.findObject` can return null. Also the function needs to something, and it returns `super.onEvent` at the end. I'll switch it around a bit though.